### DISCUSSION TITLE
Willing to try OTA Expo-updates but got an error

### DISCUSSION THREAD

**Discussion by putuoka** [infinitered/ignite#2348] [Q&A]
### Describe the bug

I'm looking easy setup OTA for react native. What i do

1. Installing ignite with npx ignite-cli@latest new TestOTA
2. Create a new account in expo https://expo.dev/accounts
3. install expo-updates with yarn add expo-updates
4. run this command `expo publish` because i have no clue what to do
5. compile APK file with this command in android folder ./gradlew assembleRelease and got an error
6. Execution failed for task ':expo-updates:compileReleaseKotlin'.
![image](https://user-images.githubusercontent.com/20880839/207506080-2c56e161-567e-4eb4-b778-4ac6bd4bb04e.png)

below step in my imagination but not executed yet because above error

7. install the compiled apk to my phone real device
8. update the ignite code in app.js by removing all code and simply add a new color box to see if it change on my phone as well or not
9. do `expo publish` again to upload the changes to expo website/cloud
10. the result is the app on my phone didn't change

am i missing something to use OTA?

### Ignite version

latest

### Additional info

```
Need to install the following packages:
  ignite-cli@8.4.3
Ok to proceed? (y) y
System
  platform           win32
  arch               x64
  cpu                4 cores      Intel(R) Core(TM) i5-4460  CPU @ 3.20GHz
  directory          IgniteExpo   D:\goodboy\contoh\IgniteExpo

JavaScript (and globally-installed packages)
  node                  16.17.0                          D:\Program Files\nodejs\node.EXE
  npm                   8.15.0                           D:\Program Files\nodejs\npm.CMD
    appcenter-cli       2.12.0
    expo-cli            6.0.8
    firebase-tools      11.14.2
    npm-check-updates   16.3.7
    yarn                1.22.19
  yarn                  1.22.19                          C:\Users\goodboy\AppData\Roaming\npm\yarn.CMD
    create-expo-app     1.1.4
  pnpm                  -                                not installed
  expo                  0.0.1 D:\goodboy\contoh\IgniteExpo   managed

Ignite
  ignite-cli         8.4.3        C:\Users\goodboy\AppData\Local\npm-cache\_npx\e31027f3785124a8\node_modules\.bin\ignite.CMD
  ignite src         build        C:\Users\goodboy\AppData\Local\npm-cache\_npx\e31027f3785124a8\node_modules\ignite-cli\build

Android
  java               -            null
  android home       -            D:\Users\goodboy\AppData\Local\Android\Sdk

Tools
  git                git version 2.30.1.windows.1   C:\Users\goodboy\AppData\Local\Programs\Git\cmd\git.EXE
```

**Comment by frankcalise** [comment]
@putuoka Hi, thanks for your interest in Ignite!

Are you trying to achieve this OTA set up via an Expo only project?

**Comment by putuoka** [comment]
> @putuoka Hi, thanks for your interest in Ignite!
> 
> Are you trying to achieve this OTA set up via an Expo only project?

I just want to test ignite with OTA so fresh install ignite with `npx ignite-cli@latest new MyProjName`
and the rest i just doing something that i don't know like installing expo-updates. to test it i need an APK file that's what i do with gradlew assembleRelease i think i need eas build apk right?

**Comment by frankcalise** [comment]
@putuoka I'll do my best to put together some steps for you.

You don't have to do anything in the android directory, that's a native dir that you don't need to deal with to go pure Expo.

You can refer to our cookbook recipe to set up your project to be a [Pristine Expo Project](https://infinitered.github.io/ignite-cookbook/docs/PristineExpoProject/) in the meantime. 

EDIT: I whipped up my initial guide at how to set up expo-updates in an Ignite Expo project. It is not released yet, but feel free to follow along and leave comments on the PR: https://github.com/infinitered/ignite-cookbook/pull/36

**Comment by frankcalise** [comment]
@putuoka any progress here? I'm going to convert this to a discussion as it's more of a how-to than issue with Ignite.
